packages/core/src/parseAmplifyOutputs.ts

```diff
@@ -333,3 +335,21 @@ function getMfaStatus(
 
 	return 'off';
 }
+
+function mapNameToBucketInfo(
```

Comment:
The function is mapping an array of buckets into an Object. 

```suggestion
function mapBucketsToObjectBucketConfig
```

Reply:
Do we do this? 🤔 I am ok with changing it to `mapNameToBucketInfoObject` but `BucketInfo` is the type we are mapping name to so this makes sense to me?

Reply:
The output of the suggested `mapNameToBucketInfo` is not `BucketInfo` right?

Reply:
Nope it is an object with `Name` as keys and `BucketInfo` as values

Reply:
maybe this makes clear sense - `mapBucketInfoToName`

Reply:
Fwiw, I think you typically map values to keys - so, in this case, literally transposing the action to the function name would probably be `mapBucketInfoToNames` - where you'd presume multiple names given the input is a list.

I do understand where @israx is coming from but it's probably a little too verbose.

Personally, I think I might have gone with something like `createBucketInfoMap` which describes what this function's actually returning - a map of BucketInfos.

Reply:
thanks chris :) makes sense , my intention was to make it clear `BucketInfo` is mapped to `name`, so i think `mapBucketInfoToName` be better. I think both make sense to me. i will make an update